usaspending_api/awards/migrations/0069_auto_20191227_1603.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2019-12-27 16:03
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('awards', '0068_auto_20191113_1134'),
```

Comment:
There is no reason to create a merge migration. Since 0069xxxx already exists, simply point this migration to depend on `0069_auto_20191226_1602`